insights/specs/default.py

```diff
@@ -105,6 +105,7 @@ def is_aws(broker):
     blkid = simple_command("/sbin/blkid -c /dev/null")
     bond = glob_file("/proc/net/bonding/bond*")
     bond_dynamic_lb = glob_file("/sys/class/net/bond[0-9]*/bonding/tlb_dynamic_lb")
+    boot_loader_entries = glob_file("/boot/loader/entries/*.conf")
```

Comment:
@xiangce @bfahr  It seems to me that `boot_loader_entries` is not collected via uploader jsons and as a result, the requirements of the GrubConf combiner are not met and the combiner and anything that depends on it doesn't run.

It seems so when I look at https://github.com/RedHatInsights/insights-core-assets/tree/uploader_json and when I try to generate local tar.gz archives and run them in a freshly built dev environment with the fresh insights-core 3.0.

Reply:
@jsvob - Since the related PR to add the spec is still not merged, see: https://github.com/RedHatInsights/insights-core-assets/pull/223